UN-OCHA/ghi-site
# GHI-20: CD v3 4.0.5 + subtheme upgrades

# GHI-20

Upgrading CD v3 to 4.0.5 and upgrading subtheme as well.

## Testing

- Check out branch, import config, which is all block placements at this time.
- The theme should look more or less identical to our [CD Demo](https://demo.commondesign-unocha-org.ahconu.org/), i.e. there is no specific brand yet and it looks like the defaults. We will wait for an official design before making further mods to the subtheme on that front.
- Look out for OCHA Services appearing in the upper-left. I had some trouble with that and I want to make sure I got it right.
- Make a change to the subtheme's Sass files and see if you can trip the linter.
